src/components/DateRangePickerInput.jsx

```diff
@@ -45,6 +45,7 @@ const propTypes = forbidExtraProps({
   readOnly: PropTypes.bool,
   showCaret: PropTypes.bool,
   showDefaultInputIcon: PropTypes.bool,
+  inputIconPosition: PropTypes.oneOf('before', 'after'),
```

Comment:
this should come from the shape, so that it doesn't have to be defined more than once